.github/PULL_REQUEST_TEMPLATE/pull-request--implemented-issue.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Pull Request: Implemented issue'
+about: Propose an implementation to help us resolve an open issue
+title: Add a relevant PR title here. Mension issue #number
+labels: ''
+assignees: ''
+
+---
+
+ **PR impact:**
+List of related issues/PRs this will solve:
+
+Impact                   | PR
+------------------------ | ------
+Relevant issue           | #number
+Other issues this solves | None or #number(s)
+Other PRs this solves    | None or #number(s)
+
+**Proposed fix:**
+A short description of what fix this is. And how it fixed the problem.
+
+ **Additional context**
+Add any other context about the solution here. Did you test the sollution on all (relevant) platforms?
```

Comment:
sollution -> solution

Reply:
Fixed typo in https://github.com/TheCherno/Hazel/pull/80/commits/bda4f8f9e22f8887fb4b046c185349f9863368a4.